src/main/resources/schema.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE products
```

Comment:
h2 DB니 크게 문제는 없을 것 같기는하지만 가능하면
이미 해당 테이블이 있는 경우에대한 DROP 문을 넣어주면 항상 안전하게 애플리케이션을 수행할 수 있을 것 같아요. 😃

```suggestion
DROP TABLE IF EXISTS product;

CREATE TABLE products
```

Reply:
H2 데이터베이스는 킬 때마다 초기화 되니깐 저걸 안넣어도 되지만 다른 DB는 킬 때마다 초기화가 자동으로 안되고 이전 DB가 계속 돌아가니깐 문제가 될 수 있으니, 저걸 습관들이라는 것이군요! 이해했습니다